po/de.po

```diff
@@ -11481,7 +11481,7 @@ msgstr "System Sprache"
 
 #: src/prefs/prefs_page_divers.c:621
 msgid "File: "
-msgstr "Dateiname:"
+msgstr "Dateiname :"
```

Comment:
Maybe a space is missing at the end since it is present in the English text.
I propose `"Dateiname : "`